packages/web/src/scss/components/Alert/_Alert.scss

```diff
@@ -25,9 +25,9 @@
 
 @include dictionaries.generate-colors(
     theme.$color-dictionary,
+    'Alert',
     (
-        default: 'color',
+        default-background: 'color',
         background: 'background-color',
```

Comment:
This matrix makes me a little bit confused, to be honest 🙂

Reply:
The problem is this background color. There are tokens called `$emotion-success-background` and `$emotion-success-default` the former is used as background when the color is the same emotion. The latter is used when the color is a different color. 

I am open to any suggestions to name this better.